src/service/xhr-impl.js

```diff
@@ -409,6 +409,7 @@ export function assertSuccess(response) {
     if (response.status < 200 || response.status >= 300) {
       /** @const {!Error} */
       const err = user().createError(`HTTP error ${response.status}`);
+      err.response = response;
```

Comment:
Do you use it anywhere? It's an interesting idea, but if there's some specific need - it might be better to just parse it out and fill it in here specifically rather leave it up to the client to dig in the XHR response?